### ISSUE TITLE
UC6 - Ontology recommendation animal models

### ISSUE THREAD

**Issue by mcourtot** [FAIRplus/FAIRPlus_squad2#56]
We’ve been asked to provide some guidance on how to describe animal models used in an in vivo study. 
We’ve thought about different approaches ranging from a big compositional model to simply collecting the terms for each component part, for example:
A mouse (species) with a particular strain (could be described by MGI nomenclature: http://www.informatics.jax.org/mgihome/nomen/ and normalizing to other terminologies like “gene” modified, “disease” represented by the model and an observed “phenotype”

Simple would be best, we would prefer a simple “minimal” standard (or set of standards) that many people could use across many different in vivo labs.

Owner: Dorothy Reilly (dorothy.reilly@novartis.com)
